### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ysam21**
Added Verida protocol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,6 +194,18 @@ Feel free to open a PR to add/update an entry in the registry!
 
 <br />
 
+## Verida Protocol
+
+### Verida Badges
+
+**Implementation**: Modified ERC-721
```
Comment (ra-phael) [inline README.md]:
```suggestion
**Implementation**: Modified ERC721 + ERC5192
```

**Review by ra-phael — COMMENTED** [review]
Thank you for your contribution!
I quickly looked at your repo and saw you also implemented EIP-5192. I'll dive into your implementation more as it looks super interesting how you designed your tokens. :)

**Comment by ysam21** [comment]
Thanks for the interest @ra-phael, feel free to reach out if you have any questions or would like to know more.

**Review by ysam21 — COMMENTED** [review]
Updated implementation details

**Review by ra-phael — APPROVED** [review]

**Pull request merged**